packages/lang/locales/en.json

```diff
@@ -572,6 +573,11 @@
   "quoteDraft.config.phoneNumber": "Phone number",
   "quoteDraft.searchProduct.addToQuoteButton": "Add to quote",
   "quoteDraft.submission.response.button.text": "OK",
+  "quoteDraft.quoteTable.oos.tip": "Insufficient stock",
```

Comment:
I would suggest be more descriptive with the translation variables, instead of using `oos` let's add out of stock so anyone understand what this means. `quoteDraft.quoteTable.outOfStock.tip`. Same for the other added variables.